tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -1,65 +1,63 @@
 import utils from 'util'
 import * as contents from './data/taquito-contents'
+import type { ExecException } from "child_process"
 const exec = utils.promisify(exec1)
 
 describe("E2E Testing for taqueria taquito plugin",  () => {
 
     const tezos = new TezosToolkit('https://hangzhounet.api.tez.ie');
     const taqueriaProjectPath = 'e2e/auto-test-taquito-plugin';
-    const contractRegex = new RegExp("^\\w{36}$");
+    const contractRegex = new RegExp(/(KT1)+\w{33}?/);
     let environment: string;
 
     beforeAll(async () => {
-        await generateTestProject(taqueriaProjectPath, ["taquito"], true);
+        await generateTestProject(taqueriaProjectPath, ["taquito"]);
     })
 
     // TODO: Consider in future to use keygen service to update account balance programmatically
     // https://github.com/ecadlabs/taqueria/issues/378
-    test.skip('Verify that taqueria taquito plugin can deploy one contract using deploy command', async () => {
+    test('Verify that taqueria taquito plugin can deploy one contract using deploy command', async () => {
         try {
             environment = "test";
             let smartContractHash = "";
 
             // 1. Copy config.json and michelson contract from data folder to artifacts folder under taqueria project
-            execSync(`cp e2e/data/config-taquito-test-environment.json ${taqueriaProjectPath}/.taq/config.json`);
-            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/`);
+            await exec(`cp e2e/data/config-taquito-test-environment.json ${taqueriaProjectPath}/.taq/config.json`);
+            await exec(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/`);
 
             // Sometimes running two deploy commands in a short period of time cause an issue
             // More details about the error - "counter %x already used for contract %y"
             // It is applicable only for auto-tests that run very fast
             // Uses retry mechanism to avoid test to fail
 
-            await waitForExpect(() => {
+            // 2. Run taq deploy on a selected test network described in "test" environment
+            // const deployCommand = 
+            const deployCommand = await exec(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).then(async (resp) => {
+                const deployResponse = resp.stdout.trim().split(/\r?\n/)[3]
+                await new Promise(resolve => setTimeout(resolve, 60000))
+                return deployResponse
+            })
 
-                // 2. Run taq deploy on a selected test network described in "test" environment
-                const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim().split(/\r?\n/)[3];
+            // 3. Verify that contract has been originated on the network
+            expect(deployCommand).toContain("hello-tacos.tz");
+            expect(deployCommand).toContain("hangzhounet");
```

Comment:
Should we be updating to Ithcanet now that Hangzhounet has stopped creating blocks?

Reply:
Most definitely, I am feeling that this PR is getting pretty big so I will make an issue to handle that and make it the next thing I do

Reply:
Since it changes frequently, wouldn't it be a good idea to make a macro or string constant for the blockchain network?

e.g. `export const CHAIN_NET = "Ithcanet"`

Reply:
@hu3man I decided to do the upgrade to ithaca in this so it is present and working. @jchenche I added a file called `network-info.ts` to hold this info which the tests will reference. Now we can just change that when a new protocol comes out and all tests that interact with a network will be good with it.